ControlMeasurements/Models/Measurement.cs

```diff
@@ -8,5 +8,10 @@ public class Measurement
         public PlaceType PlaceType { get; set; }
         public MeasurementType MeasurementType { get; set; }
         public int Value { get; set; }
+        public DateTime MeasurementDate { get; set; }
```

Comment:
W zasadzie to pole może się po prostu nazywać Date, ale tak tez jest spoko :)

Reply:
Przydała sie ta zmiana, bo lepiej wyglada przy zapisywaniu pomiaru.  ;)

Reply:
Pamiętaj, że zawsze możesz zmienić wyświetlaną nazwę za pomocą DataAnnotations :) chyba [Display(name="costam")] jak dobrze pamiętam

Reply:
aha oki :)